src/date-picker/index.tsx

```diff
@@ -66,14 +65,15 @@ const DatePicker = React.forwardRef(
     const [isDropDownOpen, setIsDropDownOpen] = useState<boolean>(false);
     const normalizedLocale = normalizeLocale('DatePicker', locale);
     const focusVisible = useFocusVisible();
+    const { ariaLabelledby, ariaDescribedby } = useFormFieldContext(rest);
 
     const internalInputRef = useRef<HTMLInputElement>(null);
     const buttonRef = useRef<ButtonProps.Ref>(null);
     useForwardFocus(ref, internalInputRef);
 
     const rootRef = useRef<HTMLDivElement>(null);
     const dropdownId = useUniqueId('calender');
-    const calendarDescriptionId = useUniqueId('calendar-description-');
```

Comment:
This is needed for LiveRegion? Otherwise the id of liveregion is not correct?

Reply:
Thanks, updated in the next revision